packages/trpc/routers/admin.ts

```diff
@@ -59,7 +69,21 @@ export const adminAppRouter = router({
       ),
     );
   }),
-
+  recrawlFailedLinks: adminProcedure.mutation(async ({ ctx }) => {
```

Comment:
It seems that we can easily add a `onlyFailures` in the existing `recrawlAllLinks` instead of creating a new endpoint?